### PR TITLE
fix accessory width in caption cells - closes #189

### PR DESCRIPTION
**Pull request by dwery**
fix for #189

### CODE REVIEW

**Comment by jwang** [comment]
seems reasonable. Let's slate this for upcoming release 1.0.11

**Comment by dwery** [comment]
thanks!

**Comment by phillipuniverse** [comment]
This has already been started by Jeff in #527.  Not sure if there's going to be any more development on it, but there are a few other references as well.  Also, this all of the table cells.  This is a quick and dirty fix for the caption cells, but should probably be further up in the class hierarchy.

File: src/Three20UI/Sources/TTTableCaptionItemCell.m [hunk 1]
```diff
@@ -76,6 +76,9 @@ static const CGFloat kKeyWidth = 75.0f;
   CGFloat margin = [tableView tableCellMargin];
   CGFloat width = tableView.width - (kKeyWidth + kKeySpacing + kTableCellHPadding*2 + margin*2);
 
+  if (item.URL)
+    width -= kAccessoryWidth;
```
Comment (phillipuniverse) [inline src/Three20UI/Sources/TTTableCaptionItemCell.m:80]: All control statements should be surrounded with brackets even if they're just 1 line.  It helps with readability and can help prevent future problems.

**Comment by dwery** [comment]
I guess Jeff's patch can be adopted instead of this one.

**Comment by dwery** [comment]
it seems #527 hasn't been merged yet.. any particular problem with that one?

**Comment by jamesgpearce** [comment]
Facebook has not maintained or supported Three20 for some time, and we are closing its old and outstanding pull requests.

Many, many thanks for your support of the project. If you have any further questions, please don't hesitate to let me know.

**Pull request closed without merging**